docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Participating in the Athens Community"
+date: 2018-02-11T15:57:56-05:00
+---
+
+Absolutely everyone is welcome to join our community at any time! We
+are a friendly and inclusive group and we'd love to have you. We have three roles in the Athens community:
+
+- Community member
+- Contributor
+- Maintainer
+
+Read on to find out more!
+
+# Community Members
+
+Community members are folks who decide they want to get involved with our
+community. Absolutely anyone can do that whenever they want. If you want
+to get involved, that doesn't mean you have to continue being involved, but
+we hope our community is welcoming and the work is interesting enough to 
+convince you to stay :)
+
+We'll provide all the support we can possibly provide to help you contribute
+in any way you'd like. If you're considering joining us, here are some ideas
+for how you can get involved:
+
+- Comment on an [issue](https://github.com/gomods/athens/issues) that you're
+interested in
+- Submit a [pull request](https://github.com/gomods/athens/pulls) (PR) to fix 
+an issue, or to improve something that doesn't have an issue
+- Review a PR that you're interested in
+- Join us at a weekly [development meeting](https://docs.google.com/document/d/1xpvgmR1Fq4iy1j975Tb4H_XjeXUQUOAvn0FximUzvIk/edit#)
+(or more than one!)
+    - See [here](https://www.youtube.com/playlist?list=PLAk08AWjk5sekD-FRjU4VVe97nltUyZ4W) for recordings of all our past meetings
+- Come chat with us in the [gophers slack](https://invite.slack.golangbridge.org/) in the `#athens` channel
+- ... and anything else that's appropriate for you!
+
+# Contributors
+
+As you participate in the community more and more, you'll have the opportunity 
+to become a contributor. Here's what being a contributor means, and what you
+should do to become one.
+
+## What Being a Contributor Means
+
+Contributors have read access to the Athens repository on Github. This means that as a contributor, you'll have issues assigned to you and you'll be requested 
+to review pull requests (PRs) via the 
+[Github pull request review system](https://help.github.com/articles/about-pull-request-reviews/).
+
+We rely heavily on the Github PR review system, which means that if you review a PR as a contributor, you can decide 
+when that PR is ready to be merged.
```

Comment:
I think this should be worded differently, currently the requirement of one approving review for merge is for those with `write` access. So a contributor could not actually be the only decision maker for a PR.

Reply:
oh really? I didn't know that!

Reply:
Agreed, due to how GH works we should change this to "you can review a PR as a contributor, and don't worry that you don't know enough, the final approval and merging is done by a maintainer.", or something like that.